### ISSUE TITLE
UnpackPauseQueue doesn't work properly

### ISSUE THREAD

**Issue by Bec-de-Xorbin** [nzbgetcom/nzbget#349]
### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### NZBGet Version

v24.3-testing

### Platform

All

### Environment

```markdown
LinkStation LS220D
armhf
```

### Current Behavior

UnpackPauseQueue doesn't work properly. It keeps on downloading the queue records when the first record of the queue is still unpacking and only stops when the first is unpacked and starts unpacking the second.

### Expected Behavior

Pause downloading when unpack is in progress no matter what.

**Comment by dnzbk** [comment]
Tested and everything looks fine to me.
Are you sure that the download continues during unpacking and not during Par2 checking/repairing?
There is an option ParPauseQueue in SETTINGS→CHECK AND REPAIR
NZBGet has a postprocessor task queue that looks like this:
Par2 → Unpacking → other scripts.

**Comment by Bec-de-Xorbin** [comment]
Yup. ParPauseQueue, UnpackPauseQueue and ScriptPauseQueue are all enabled.
![nzbget](https://github.com/user-attachments/assets/a8003431-211e-4989-acfb-5a68cd50f1cf)
